app/models/organisation.rb

```diff
@@ -156,16 +157,20 @@ def editable_duplicate_sales_logs_sets
     sales_logs.after_date(FormHandler.instance.sales_earliest_open_for_editing_collection_start_date).duplicate_sets.map { |array_str| array_str ? array_str.map(&:to_i) : [] }
   end
 
-  def recently_absorbed_organisations_grouped_by_merge_date
-    return unless absorbed_organisations.present? && absorbed_organisations.merged_during_open_collection_period.present?
+  def organisations_absorbed_during_displayed_collection_period_grouped_by_merge_date
+    return unless absorbed_organisations.present? && absorbed_organisations.merged_during_displayed_collection_period.present?
 
-    absorbed_organisations.merged_during_open_collection_period.group_by(&:merge_date)
+    absorbed_organisations.merged_during_displayed_collection_period.group_by(&:merge_date)
   end
 
   def has_recent_absorbed_organisations?
     absorbed_organisations&.merged_during_open_collection_period.present?
   end
 
+  def has_organisations_absorbed_during_displayed_collection_period?
+    absorbed_organisations&.merged_during_displayed_collection_period.present?
```

Comment:
as above since `absorbed_organisations` is an association and `merged_during_...` then I feel like the former can't be nil, so the safe operator isn't necessary